### PR TITLE
Fix UTF8 handling

### PR DESCRIPTION
**Pull request by strk**
This is what we want for proper upload of BDE files...

### CODE REVIEW

**Comment by imincik** [comment]
@strk , running with this patch only for `crs_transact_type` table, this is the output 

```
linz_bde_uploader: 2019/07/16 08:47:14 WARN> BdeDatabase.pm:756 - SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 08:47:14 INFO> BdeDatabase.pm:384 - Job 2362 created
linz_bde_uploader: 2019/07/16 08:47:14 INFO> BdeUpload.pm:1122 - Running start hook: /opt/bde-processor/bde_update_notify bde_start 2362
linz_bde_uploader: 2019/07/16 08:47:14 SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 08:47:14 Job 2362 created
linz_bde_uploader: 2019/07/16 08:47:14 Running start hook: /opt/bde-processor/bde_update_notify bde_start 2362
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeUpload.pm:1126 - Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "714ce9dc-6bf1-5aa6-a113-bc7a8365de47"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeUpload.pm:746 - Applying 20190706214900 level 0 update (job 2362)
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeUpload.pm:915 - Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeUpload.pm:1017 - Loading file xtrt from dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "714ce9dc-6bf1-5aa6-a113-bc7a8365de47"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 08:47:15 Applying 20190706214900 level 0 update (job 2362)
linz_bde_uploader: 2019/07/16 08:47:15 Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 Loading file xtrt from dataset 20190706214900
linz_bde_uploader: NOTICE: using built-in stdout support in bde_copy 1.3.0 unknown
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeUpload.pm:979 - Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Setting up L0 update
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Copying schema information to temp table
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Applying table differences
linz_bde_uploader: 2019/07/16 08:47:15 WARN> BdeDatabase.pm:756 - level 0 incremental applied 1 updates for crs_transact_type in 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Checking table count
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Recording the upload
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Table crs_transact_type completed: 0 insertions, 1 updates, 0 deletions
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Completing dataset revision for dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:756 - Finished completing dataset revision for dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 INFO> BdeDatabase.pm:436 - Job 2362 finished successfully
linz_bde_uploader: 2019/07/16 08:47:15 Applying level 0 update 20190706214900 into table crs_transact_type
e_update_notify bde_finish 2362
linz_bde_uploader: 2019/07/16 08:47:15 Setting up L0 update
linz_bde_uploader: 2019/07/16 08:47:15 Copying schema information to temp table
linz_bde_uploader: 2019/07/16 08:47:15 Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 08:47:15 Applying table differences
linz_bde_uploader: 2019/07/16 08:47:15 level 0 incremental applied 1 updates for crs_transact_type in 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 Checking table count
linz_bde_uploader: 2019/07/16 08:47:15 Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 08:47:15 Recording the upload
linz_bde_uploader: 2019/07/16 08:47:15 Table crs_transact_type completed: 0 insertions, 1 updates, 0 deletions
linz_bde_uploader: 2019/07/16 08:47:15 Completing dataset revision for dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 Finished completing dataset revision for dataset 20190706214900
linz_bde_uploader: 2019/07/16 08:47:15 Job 2362 finished successfully
linz_bde_uploader: 2019/07/16 08:47:15 Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2362
linz_bde_uploader: 2019/07/16 08:47:16 INFO> BdeUpload.pm:1126 - Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "d7c28d7c-4676-5e28-ae6e-d8daaee39cca"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 08:47:16 INFO> linz_bde_uploader:250 - Duration of job: 00:00:02
linz_bde_uploader: 2019/07/16 08:47:16 Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "d7c28d7c-4676-5e28-ae6e-d8daaee39cca"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 08:47:16 Duration of job: 00:00:02
BDE Processor finished successfully at 2019-07-16T08:47:16+1200

```

**Comment by imincik** [comment]
`linz_bde_uploader: 2019/07/16 08:47:15 WARN> BdeDatabase.pm:756 - level 0 incremental applied 1 updates for crs_transact_type in 20190706214900` shouldn't be here if patch is working.

File: lib/LINZ/BdeDatabase.pm [hunk 1]
```diff
@@ -541,6 +541,7 @@ sub streamDataToTempTable
 
     # See https://metacpan.org/pod/DBD::Pg#COPY-support
     my @buf;
+    $datafh->binmode(":encoding(UTF-8)");
```
Comment (palmerj) [inline lib/LINZ/BdeDatabase.pm]: Would setting this when the FH is first opened be a better place? e.g in BDEUpload-> _OpenDataFile https://github.com/linz/linz-bde-uploader/blob/master/lib/LINZ/BdeUpload.pm#L1193

Comment (strk) [reply lib/LINZ/BdeDatabase.pm]: I'm still thinking if if isn't better to only decode when passing the value to pg_copydata instead.
And... I've to check why the test is now failing..

**Comment by imincik** [comment]
Looks ok now:

```
linz_bde_uploader: 2019/07/16 22:17:18 WARN> BdeDatabase.pm:757 - SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:17:18 INFO> BdeDatabase.pm:384 - Job 2361 created
linz_bde_uploader: 2019/07/16 22:17:18 INFO> BdeUpload.pm:1122 - Running start hook: /opt/bde-processor/bde_update_notify bde_start 2361
linz_bde_uploader: 2019/07/16 22:17:18 SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:17:18 Job 2361 created
linz_bde_uploader: 2019/07/16 22:17:18 Running start hook: /opt/bde-processor/bde_update_notify bde_start 2361
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeUpload.pm:1126 - Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "56e7c5bc-0f9b-5cd1-96ec-901ba9f58014"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeUpload.pm:746 - Applying 20190706214900 level 0 update (job 2361)
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeUpload.pm:915 - Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeUpload.pm:1017 - Loading file xtrt from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:17:19 Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "56e7c5bc-0f9b-5cd1-96ec-901ba9f58014"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:17:19 Applying 20190706214900 level 0 update (job 2361)
linz_bde_uploader: 2019/07/16 22:17:19 Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:17:19 Loading file xtrt from dataset 20190706214900
linz_bde_uploader: NOTICE: using built-in stdout support in bde_copy 1.3.0 unknown
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeUpload.pm:979 - Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Setting up L0 update
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Applying table differences
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 INFO> BdeDatabase.pm:757 - Recording the upload
linz_bde_uploader: 2019/07/16 22:17:20 INFO> BdeDatabase.pm:757 - Table crs_transact_type completed: 0 insertions, 0 updates, 0 deletions
linz_bde_uploader: commit ineffective with AutoCommit enabled at /usr/share/perl5/LINZ/BdeDatabase.pm line 786.
linz_bde_uploader: 2019/07/16 22:17:20 INFO> BdeDatabase.pm:436 - Job 2361 finished successfully
linz_bde_uploader: 2019/07/16 22:17:20 INFO> BdeUpload.pm:1122 - Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2361
linz_bde_uploader: 2019/07/16 22:17:19 Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 Setting up L0 update
linz_bde_uploader: 2019/07/16 22:17:19 Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:17:19 Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 Applying table differences
linz_bde_uploader: 2019/07/16 22:17:19 Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:17:19 Recording the upload
linz_bde_uploader: 2019/07/16 22:17:20 Table crs_transact_type completed: 0 insertions, 0 updates, 0 deletions
linz_bde_uploader: 2019/07/16 22:17:20 Job 2361 finished successfully
linz_bde_uploader: 2019/07/16 22:17:20 Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2361
linz_bde_uploader: 2019/07/16 22:17:21 INFO> BdeUpload.pm:1126 - Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "5343fa73-b292-5f6b-9028-afd414e2a5ed"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:17:21 INFO> linz_bde_uploader:250 - Duration of job: 00:00:03
linz_bde_uploader: 2019/07/16 22:17:21 Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "5343fa73-b292-5f6b-9028-afd414e2a5ed"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:17:21 Duration of job: 00:00:03
BDE Processor finished successfully at 2019-07-16T22:17:21+1200
```

**Comment by imincik** [comment]
Without this patch we are receiving incorrect behavior:

```
linz_bde_uploader: 2019/07/16 22:22:59 WARN> BdeDatabase.pm:757 - SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:22:59 INFO> BdeDatabase.pm:384 - Job 2367 created
linz_bde_uploader: 2019/07/16 22:22:59 INFO> BdeUpload.pm:1122 - Running start hook: /opt/bde-processor/bde_update_notify bde_start 2367
linz_bde_uploader: 2019/07/16 22:22:59 SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:22:59 Job 2367 created
linz_bde_uploader: 2019/07/16 22:22:59 Running start hook: /opt/bde-processor/bde_update_notify bde_start 2367
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:1126 - Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "d2fd371d-9733-525f-ad1e-62ba30f0cbc7"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:746 - Applying 20190706214900 level 0 update (job 2367)
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:915 - Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:1017 - Loading file xtrt from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:23:00 Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "d2fd371d-9733-525f-ad1e-62ba30f0cbc7"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:23:00 Applying 20190706214900 level 0 update (job 2367)
linz_bde_uploader: 2019/07/16 22:23:00 Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:23:00 Loading file xtrt from dataset 20190706214900
linz_bde_uploader: NOTICE: using built-in stdout support in bde_copy 1.3.0 unknown
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:979 - Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Setting up L0 update
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Applying table differences
linz_bde_uploader: 2019/07/16 22:23:00 WARN> BdeDatabase.pm:757 - level 0 incremental applied 1 updates for crs_transact_type in 20190706214900
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Checking table count
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Recording the upload
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:757 - Table crs_transact_type completed: 0 insertions, 1 updates, 0 deletions
linz_bde_uploader: commit ineffective with AutoCommit enabled at /usr/share/perl5/LINZ/BdeDatabase.pm line 786.
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeDatabase.pm:436 - Job 2367 finished successfully
linz_bde_uploader: 2019/07/16 22:23:00 INFO> BdeUpload.pm:1122 - Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2367
linz_bde_uploader: 2019/07/16 22:23:00 Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 Setting up L0 update
linz_bde_uploader: 2019/07/16 22:23:00 Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:23:00 Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 Applying table differences
linz_bde_uploader: 2019/07/16 22:23:00 level 0 incremental applied 1 updates for crs_transact_type in 20190706214900
linz_bde_uploader: 2019/07/16 22:23:00 Checking table count
linz_bde_uploader: 2019/07/16 22:23:00 Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:23:00 Recording the upload
linz_bde_uploader: 2019/07/16 22:23:00 Table crs_transact_type completed: 0 insertions, 1 updates, 0 deletions
linz_bde_uploader: 2019/07/16 22:23:00 Job 2367 finished successfully
linz_bde_uploader: 2019/07/16 22:23:00 Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2367
linz_bde_uploader: 2019/07/16 22:23:01 INFO> BdeUpload.pm:1126 - Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "7453c9ef-e8d8-51fb-9deb-9c484f1c3262"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:23:01 INFO> linz_bde_uploader:250 - Duration of job: 00:00:02
linz_bde_uploader: 2019/07/16 22:23:01 Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "7453c9ef-e8d8-51fb-9deb-9c484f1c3262"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:23:01 Duration of job: 00:00:02
BDE Processor finished successfully at 2019-07-16T22:23:01+1200

```

**Comment by imincik** [comment]
@strk , no issues even with another change you did in this PR

```
linz_bde_uploader: 2019/07/16 22:39:51 WARN> BdeDatabase.pm:756 - SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:39:51 INFO> BdeDatabase.pm:384 - Job 2369 created
linz_bde_uploader: 2019/07/16 22:39:51 INFO> BdeUpload.pm:1122 - Running start hook: /opt/bde-processor/bde_update_notify bde_start 2369
linz_bde_uploader: 2019/07/16 22:39:51 SET LOCAL can only be used in transaction blocks
linz_bde_uploader: 2019/07/16 22:39:51 Job 2369 created
linz_bde_uploader: 2019/07/16 22:39:51 Running start hook: /opt/bde-processor/bde_update_notify bde_start 2369
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:1126 - Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "c113c7e2-1d73-594f-95c4-9245b6cb978b"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:746 - Applying 20190706214900 level 0 update (job 2369)
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:915 - Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:1017 - Loading file xtrt from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:39:52 Event start hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "c113c7e2-1d73-594f-95c4-9245b6cb978b"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:39:52 Applying 20190706214900 level 0 update (job 2369)
linz_bde_uploader: 2019/07/16 22:39:52 Loading crs_transact_type from dataset 20190706214900
linz_bde_uploader: 2019/07/16 22:39:52 Loading file xtrt from dataset 20190706214900
linz_bde_uploader: NOTICE: using built-in stdout support in bde_copy 1.3.0 unknown
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:979 - Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Setting up L0 update
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Applying table differences
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Recording the upload
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:756 - Table crs_transact_type completed: 0 insertions, 0 updates, 0 deletions
linz_bde_uploader: commit ineffective with AutoCommit enabled at /usr/share/perl5/LINZ/BdeDatabase.pm line 785.
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeDatabase.pm:436 - Job 2369 finished successfully
linz_bde_uploader: 2019/07/16 22:39:52 INFO> BdeUpload.pm:1122 - Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2369
linz_bde_uploader: 2019/07/16 22:39:52 Applying level 0 update 20190706214900 into table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 Setting up L0 update
linz_bde_uploader: 2019/07/16 22:39:52 Copying schema information to temp table
linz_bde_uploader: 2019/07/16 22:39:52 Analyzing temp table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 Applying table differences
linz_bde_uploader: 2019/07/16 22:39:52 Dropping table crs_transact_type
linz_bde_uploader: 2019/07/16 22:39:52 Recording the upload
linz_bde_uploader: 2019/07/16 22:39:52 Table crs_transact_type completed: 0 insertions, 0 updates, 0 deletions
linz_bde_uploader: 2019/07/16 22:39:52 Job 2369 finished successfully
linz_bde_uploader: 2019/07/16 22:39:52 Running finish hook: /opt/bde-processor/bde_update_notify bde_finish 2369
linz_bde_uploader: 2019/07/16 22:39:53 INFO> BdeUpload.pm:1126 - Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "78878776-2ffc-5c26-866e-d2d18445cec4"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:39:53 INFO> linz_bde_uploader:250 - Duration of job: 00:00:02
linz_bde_uploader: 2019/07/16 22:39:53 Event finish hook result: 0
linz_bde_uploader: {
linz_bde_uploader:     "MessageId": "78878776-2ffc-5c26-866e-d2d18445cec4"
linz_bde_uploader: }
linz_bde_uploader: 2019/07/16 22:39:53 Duration of job: 00:00:02
BDE Processor finished successfully at 2019-07-16T22:39:53+1200

```

**Comment by strk** [comment]
We might want this in 2.5 branch too, being an hotfix

**Comment by palmerj** [comment]
> We might want this in 2.5 branch too, being an hotfix

Yes we need to backport this fix to the production branch.

**Comment by strk** [comment]
> Yes we need to backport this fix to the production branch.

Ready for review: https://github.com/linz/linz-bde-uploader/pull/211

**Pull request merged**